frontend/javascripts/viewer/view/ai_jobs/train_ai_model/ai_training_settings.tsx

```diff
@@ -0,0 +1,78 @@
+import { SettingOutlined } from "@ant-design/icons";
+import { Card, Col, Form, Input, InputNumber, Row, Space } from "antd";
+import type { FormProps } from "antd";
+import type React from "react";
+import { APIJobType } from "types/api_types";
+
+import { useAiTrainingJobContext } from "./ai_training_job_context";
+
+export const AiTrainingSettings: React.FC = () => {
+  const {
+    modelName,
+    setModelName,
+    comments,
+    setComments,
+    selectedTask,
+    maxDistanceNm,
+    setMaxDistanceNm,
+  } = useAiTrainingJobContext();
+
+  const handleValuesChange: FormProps["onValuesChange"] = (changedValues) => {
+    if (Object.prototype.hasOwnProperty.call(changedValues, "modelName")) {
+      setModelName(changedValues.modelName);
+    }
+    if (Object.prototype.hasOwnProperty.call(changedValues, "comments")) {
+      setComments(changedValues.comments);
+    }
+    if (Object.prototype.hasOwnProperty.call(changedValues, "maxDistanceNm")) {
+      setMaxDistanceNm(changedValues.maxDistanceNm);
+    }
+  };
+
+  const formFields = [
+    { name: ["modelName"], value: modelName },
+    { name: ["comments"], value: comments },
+    { name: ["maxDistanceNm"], value: maxDistanceNm },
+  ];
+
+  return (
+    <Card
+      type="inner"
+      title={
+        <Space align="center">
+          <SettingOutlined style={{ color: "#1890ff" }} />
```

Comment:
the color should be extracted somehow. I think i've seen it multiple times.

Reply:
Done